test/test_priorityqueue.jl

```diff
@@ -93,6 +93,18 @@ while !isempty(pq)
     @test 10 != dequeue!(pq)
 end
 
+priorities2 = Dict(zip('a':'e', 5:-1:1))
+pq = PriorityQueue(priorities2)
+try
+    dequeue_pair!(pq, 'g')
+    error("should have resulted in KeyError")
```

Comment:
Yeah, that syntax confused me as well but I figured it was better to be consistent with what was there. I'll fix it.